### PR TITLE
[yamui] More reliable displaying of content over devices. Contributes: JB#14745

### PR DESCRIPTION
**Pull request by philippedeswert**

### CODE REVIEW

File: minui/graphics_fbdev.c [hunk 1]
```diff
@@ -179,6 +180,20 @@ static gr_surface fbdev_init(minui_backend* backend) {
 }
 
 static gr_surface fbdev_flip(minui_backend* backend __attribute__((unused))) {
+
+#if defined(RECOVERY_BGRA)
+        // In case of BGRA, do some byte swapping
```
Comment (thp) [inline minui/graphics_fbdev.c]: Would it be enough to just flip the R and B component inside `gr_color()` (and possibly other places that deal with color)? This way, BGRA support would come at the same cost as RGBA support, whereas with this approach, we have to scan through the whole framebuffer to swap bytes.

Comment (philippedeswert) [inline minui/graphics_fbdev.c]: Unfortunately it only seems to work with the text, not with the images. Anyway it is not needed in normal conditions, so they are there for reference if we need them.

Comment (thp) [inline minui/graphics_fbdev.c]: Oh yeah, didn't think of images. Makes sense to have it there for reference.

**Comment by thp** [comment]
LGTM.

**Pull request merged**